tools/browser-matcher/common-matchers.mjs

```diff
@@ -22,3 +22,21 @@ export const supportsFetch = new SpecMatcher()
   .include('firefox>=40')
   .include('ios>=11')
   .include('android')
+
+// Firefox <= 19 has inconsistent timing data.
```

Comment:
I think we need to settle on a minimum browser version that we simply will never test. I am pretty sure you can't test FF 19 with W3C protocol.

Reply:
I don't know what a sensible minimum is, but a 10 year old Firefox probably falls well below it. I'll remove this matcher.